app/models/concerns/issue_search.rb

```diff
@@ -0,0 +1,189 @@
+module IssueSearch
+  extend ActiveSupport::Concern
+
+  included do
+    include Elasticsearch::Model
+    include Elasticsearch::Model::Callbacks
+
+
+    FIELDS = ['full_title^2', 'exact_title^2', 'body']
+
+    settings index: { number_of_shards: 1, number_of_replicas: 0 } do
+      mapping do
+        indexes :full_title, :analyzer => 'snowball', :boost => 6
+        indexes :exact_title, :index => :not_analyzed, :boost => 2
```

Comment:
No need for the exact title on this, it's just a hack for priority in search based on exact matching